src/Authentication/Authenticators/Session.php

```diff
@@ -459,6 +508,38 @@ private function startLogin(User $user): void
         $response->noCache();
     }
 
+    private function getSessionUserInfo(): array
+    {
+        return session(setting('Auth.sessionConfig')['field']) ?? [];
+    }
+
+    /**
+     * @return int|string|null
+     */
+    private function getSessionUser(string $key)
+    {
+        $sessionUserInfo = $this->getSessionUserInfo();
+
+        return $sessionUserInfo[$key] ?? null;
+    }
+
+    /**
+     * @param int|string|null $value
+     */
+    private function setSessionUser(string $key, $value): void
```

Comment:
These two make me think they will get/set the user. What about `getSessionKey()` or `Data()` or `Field()`?

Reply:
Renamed to `getSessionKey()`.